pkg/controller/nodelifecycle/scheduler/taint_manager_test.go

```diff
@@ -499,8 +496,7 @@ func TestUpdateNode(t *testing.T) {
 			controller.recorder = testutil.NewFakeRecorder()
 			go controller.Run(ctx)
 			controller.NodeUpdated(item.oldNode, item.newNode)
-			// wait a bit
-			time.Sleep(timeForControllerToProgress)
```

Comment:
is the constant still needed?

Reply:
It is used in two places still. However, I renamed it to `timeForControllerToProgressForSanityTesting` as its only purpose now is to wait a little bit after the last action so that there is no panic.